packages/navigation/src/history.js

```diff
@@ -44,8 +49,17 @@ function getHistory() {
 			},
 			createHref: ( ...args ) =>
 				browserHistory.createHref.apply( browserHistory, args ),
-			push: ( ...args ) =>
-				browserHistory.push.apply( browserHistory, args ),
+			push( ...args ) {
+				const [ href ] = args;
```

Comment:
There _may_ be an edge case where `history.push` is called with different arguments, but I couldn't produce anything like that in testing; anyone know if this could be an issue?

Reply:
I added the above argument condition just in case.